manifests/t/Tyrrrz/LightBulb/2.3.3/Tyrrrz.LightBulb.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Tyrrrz.LightBulb
+PackageVersion: 2.3.3
+InstallerType: exe
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/Tyrrrz/LightBulb/releases/download/2.3.3/LightBulb-Installer.exe
+  InstallerSha256: C47F4E2945ED2B743EBB37ACFE517A33F256D358BAF362CF835F52E29D5DF609
+  InstallerSwitches:
+    Silent: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-
+    SilentWithProgress: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-

```

```suggestion
  InstallerSwitches:
    Custom: /NORESTART
```
Just a bit of clean up